.changeset/tall-shrimps-nail.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sigle/tailwind-style': minor
```

Comment:
For this package we can create another changelog entry, as it's separated from the app the message could just be regarding dark mode.
We are not limited by one changeset per pull request btw so a pull request can contains potentially 100 changesets :)

Reply:
Oh right I see, that's pretty useful to know :) thanks!